pipeline_dp/pipeline_backend.py

```diff
@@ -700,6 +728,14 @@ def map(self, col, fn, stage_name: typing.Optional[str] = None):
                                       chunksize=self.chunksize,
                                       **self.pool_kwargs)
 
+    def map_with_side_inputs(self,
+                             col,
+                             fn,
+                             side_input_cols,
+                             stage_name: typing.Optional[str] = None):
+        side_inputs = [list(side_input) for side_input in side_input_cols]
```

Comment:
not sure about it, whether we should materalize collections in multiproclocal this way.

Reply:
ack